srcpkgs/giac/template

```diff
@@ -14,6 +14,11 @@ homepage="https://www-fourier.ujf-grenoble.fr/~parisse/giac.html"
 distfiles="https://www-fourier.ujf-grenoble.fr/~parisse/debian/dists/stable/main/source/giac_${version%.*}-${version##*.}.tar.gz"
 checksum=bbd6f0aafe373de3c9ed53d21878224e847359fdd9a7cdf9e9096e7c960f10b4
 
+if [ "$XBPS_TARGET_LIBC" = musl ]; then
+	# need more than 4*65536 stack, see try_parse() in gen.cc line 11812
+	LDFLAGS+=" -Wl,-z,stack-size=1048576"
```

Comment:
ah, nice find! You can apply this unconditionally, it shouldn't affect the resulting glibc binary.

Reply:
I was in doubt. Should I make the stack as small as required to pass tests (~270K seems enough)?

I'd rather give it some slack to be safe.

Reply:
Oh, I meant the test for musl wasn't necessary. And you should just set `LDFLAGS`, not add to it. The value is good, though 2MB is closer to glibc's default thread stack size, if you want to go with that.